[merge 1] Add information about ephemeral storage support (#65)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_ephemeral-storage.adoc

// Module included in the following assemblies:
//
// <List assemblies here, each on a new line>

// This module can be included from assemblies using the following include statement:
// include::<path>/con_ephemeral-storage.adoc[leveloffset=+1]

// The file name and the ID are based on the module title. For example:
// * file name: con_my-concept-module-a.adoc
// * ID: [id='con_my-concept-module-a_{context}']
// * Title: = My concept module A
//
// The ID is used as an anchor for linking to the module. Avoid changing
// it after the module has been published to ensure existing links are not
// broken.
//
// The `context` attribute enables module reuse. Every module's ID includes
// {context}, which ensures that the module has a unique ID even if it is
// reused multiple times in a guide.
//
// In the title, include nouns that are used in the body text. This helps
// readers and search engines find information quickly.
// Do not start the title with a verb. See also _Wording of headings_
// in _The IBM Style Guide_.
[id="ephemeral-storage_{context}"]
= Ephemeral Storage

> Use ephemeral storage to run {ProjectShort) without persistently storing data in your {OpenShiftShort} cluster.  It is not recommended that you use ephemeral storage in a production environment due to the volatility of the data when the platform is operating correctly and as designed. For example, restarting a pod or rescheduling the workload to another node results in the loss of any local data written since the pod started.

Author: ```suggestion
Use ephemeral storage to run {ProjectShort) without persistently storing data in your {OpenShiftShort} cluster.  Ephemeral storage is not recommended in a production environment due to the volatility of the data in the platform when operating correctly and as designed. For example, restarting a pod or rescheduling the workload to another node results in the loss of any local data written since the pod started.
```

Author: @JoanneOFlynn2018 further tweaked

> Ephemeral storage allows running {ProjectShort} without persistently storing the data in your {OpenShiftShort} cluster. Using ephemeral storage is not recommended for production environments due to the volatility of the data when the platform is operating correctly and as designed. For example, restarting a pod or rescheduling the workload to another node would result in the loss of any local data written since the pod started.

Reviewer: Removing anthropomorphism at the start of the par. 

```suggestion
Use ephemeral storage to run {ProjectShort) without persistently storing data in your {OpenShiftShort} cluster.  It is not recommended that you use ephemeral storage in a production environment due to the volatility of the data when the platform is operating correctly and as designed. For example, restarting a pod or rescheduling the workload to another node results in the loss of any local data written since the pod started.
```
